tgui/packages/tgui/interfaces/SpawnersMenu.js

```diff
@@ -27,7 +27,11 @@ export const SpawnersMenu = (props, context) => {
                       </Flex.Item>
                     )}
                     <Flex.Item fontSize="14px" color="green" bold mt={0.4} mr={2}>
-                      Мест: {spawner.amount_left}
+                      Мест: {
+                        spawner.infinity
```

Comment:
```suggestion
                        spawner.is_infinite
```
В дате ис_инфините передается